Aztec/Classes/Formatters/Implementations/LinkFormatter.swift

```diff
@@ -21,15 +24,27 @@ class LinkFormatter: StandardAttributeFormatter {
                }
             } else {
                 attributeValue = NSURL(string: "")!
-            }            
+            }
+            finalRepresentation = representation
         } else {
 
             // There's no support fora link representation that's not an HTML element, so this
             // scenario should only be possible if `representation == nil`.
             //
             assert(representation == nil)
+            var attributes = [Attribute]()
```

Comment:
Readability: Add some spacing around the code below.  Right now it's a bit too tight.